packages/xo-web/src/common/ip-utils.js

```diff
@@ -99,24 +104,8 @@ export const formatIps = ips => {
       (splitIp1[0] - splitIp2[0]) * 256 * 256 * 256
     )
   })
-  const range = { first: '', last: '' }
-  const formattedIps = []
-  let index = 0
-  forEach(sortedIps, ip => {
-    if (ip !== getNextIpV4(range.last)) {
-      if (range.first) {
-        formattedIps[index] =
-          range.first === range.last ? range.first : { ...range }
-        index++
-      }
-      range.first = range.last = ip
-    } else {
-      range.last = ip
-    }
-  })
-  formattedIps[index] = range.first === range.last ? range.first : range
 
-  return formattedIps
+  return [{ first: head(sortedIps), last: last(sortedIps) }]
```

Comment:
We need to show all the ranges, not just a `<smallest IP> → <biggest IP>` range.

Reply:
In this case, the previous range formatting is correct !

Reply:
No it's not because a range such as `1.1.1.42-1.1.2.42` should not be split into 2 ranges.